tutorial/practicals/LFRic/building_code/3_time_evolution/configuration.nml

```diff
@@ -1,5 +1,5 @@
 &base_mesh
-  filename        = 'mesh_planar100x100-1000x1000.nc'
+  filename        = '../mesh_planar100x100-1000x1000.nc'
```

Comment:
That confused me, since I was sure I had tested that several time (this branch is merged into my makefile-cleanup). But I just now realised that you build the solution in a separate directory (which had the proper path). I have changed the makefiles for the psydata stuff to compile in the same directory (using the modified files in solutions). So all good, I have an explanation :)

Reply:
Explanation is good, but this still needs to point to the `data` directory, similar how it is done in `3_time_evolution/solution/configuration.nml` :) There is `../../data/mesh_planar100x100-1000x1000.nc` and here it should be `../data/mesh_planar100x100-1000x1000.nc` as it is one level up.

Reply:
That drove me nuts, since I was sure I had done that. Turns out that I tackled the review in a different checked-out version of the repo, and had two changes still sitting there uncommitted - I just forgot to add them *sigh* This time it's done ... really. Sorry for that :( I also added the other change in (made the code compatible with older jinja versions).